changelogs/2.11.3.md

```diff
@@ -0,0 +1,161 @@
+# 2.11.3
+
+Date: 2024-04-17
+Tag: 2.11.3
+
+## Overview
+
+2.11.3 is the next stable release in the [long-term support (LTS)
+version][release_policy][release_policy] 2.11.x release series.
+
+The label "stable" means there are 1.10.x-based applications running in
+production for quite a while without known crashes, incorrect results or
+other showstopper bugs.
+
+This release introduces 1 improvements and resolves roughly 54
+issues since the 2.11.2 version.
```

Comment:
This '1 improvements' is about tools and I think we'll have only bugfixes here after moving it to a top-level `## Tools` section.

Reply:
done